Monika After Story/game/script-moods.rpy

```diff
@@ -249,6 +249,30 @@ label mas_mood_angry:
     m "Remember: A happy [player] makes a happy Monika!"
     return
     
+init 5 python:
+    addEvent(Event(persistent._mas_mood_database,"mas_mood_scared",prompt="anxious",category=[store.mas_moods.TYPE_NEUTRAL],unlocked=True),eventdb=store.mas_moods.mood_db)
+
+label mas_mood_scared:
+    m 1g "[player], are you alright?"
+    m 1f "It's worrying for me to see you're suddenly getting anxious..."
+    m 1m "I wish I could lend you myself to comfort you at this time..."
+    m 3e "But I can help you to calm yourselves down, [player]."
+    m 3a "Do you remember when I talked to you about confidence?"
+    m 3b "Keeping up your appearance does help on faking your own confidence."
+    m "And to do so, you need to maintain your heart rate by taking deep breath until you calm down."
+    m "Maybe you could take things slowly, and do it one at a time."
+    m "You'd be surprised on how smooth can you be if you let the time flows on it's own."
+    m 4b "You can even spend those few minutes to meditate!"
```

Comment:
"those" -> "a"
Just sounds weird to read, but that one in particular is a suggestion